.github/labeler.yml

```diff
@@ -1,3 +1,3 @@
 ---
 skip-changelog:
- - head-branch: ['^release-*', 'release']
+ - head-branch: ['^release-*']
```

Comment:
shouldn't we keep the `'release'` branch name?

Reply:
it's more like a pattern, afaik. it matches on `release`, `prep-release`, `donotreleaseme`. for the containers i want to encourage the people to use the `release-vX.Y.Z` style. like it is mentioned in the RELEASE.md

but i can add it back, i have no strong feelings about it.

Reply:
sounds good to me, I think we can also leave it out.